services/src/main/java/org/fao/geonet/api/site/SiteInformation.java

```diff
@@ -89,24 +93,37 @@ public void setDatabaseProperties(HashMap<String, String> databaseProperties) {
         this.databaseProperties = databaseProperties;
     }
 
+    @JsonProperty(value = "version")
+    public HashMap<String, String> getVersionProperties() {
```

Comment:
Does really need properties to be a HashMap ? Is Map sufficient ? Is this case do we need to recopy around line 234 ?